frontend/src/app/shared/components/fields/edit/field-types/combined-date-edit-field.component.ts

```diff
@@ -35,6 +35,8 @@ import { OpModalComponent } from 'core-app/shared/components/modal/modal.compone
 import { DatePickerModalComponent } from 'core-app/shared/components/datepicker/datepicker.modal';
 import { TimezoneService } from 'core-app/core/datetime/timezone.service';
 
+export const combinedDateFieldClass = 'combinedDate';
```

Comment:
Feels a bit weird to have this exported here and only used in one other place, but I guess it's less weird than a magic string in a file which has nothing to do with it directly.

Reply:
Yeah, I also wanted to avoid a magic `combinedDate` in the datepicker. Here, it is at least at the right spot. 🤷‍♀️